### PR TITLE
Mgmt 3598 update cluster for kubeapi

### PR DESCRIPTION
**Pull request by RazRegev**

### CODE REVIEW

**Comment by RazRegev** [comment]
/hold
based on https://github.com/openshift/assisted-service/pull/1162

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -1882,6 +1882,47 @@ func (b *bareMetalInventory) updateDhcpNetworkParams(updates map[string]interfac
 	return nil
 }
 
+func (b *bareMetalInventory) updateKubeAPIClusterNetworkParams(
```
Comment (filanov) [inline internal/bminventory/inventory.go:1885]: I still don't understand why do we need a specific API to update kube-api?

**Comment by RazRegev** [comment]
/close, 
no need for this PR, validations will be at hive side

**Pull request closed without merging**